### PR TITLE
Milestone 2: Store, Components and Actions were implemented

### PR DESCRIPTION
**Pull request by carlosprietobarron**
Create components, actions, and store

Connect Store with Reducers

Create initial state for reducers

Validate props and states

### CODE REVIEW

File: src/component/BookList.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+BookList.propTypes = {
+  books: propTypes.shape(propTypes.shape({
+    id: propTypes.number,
+    title: propTypes.string,
+    category: propTypes.string,
+  }).isRequired).isRequired,
+};
+
+export default connect(mapStateToProps)(BookList);
```
Comment (BrittanyBlake) [inline src/component/BookList.js]: - [ ] The  `BookList` component should be placed in the `src/containers` directory.

File: src/component/BookForm.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      <input type="text" id="title" />
+      <select value="Action" id="category">
+        {categories.map(MakeCategory)}
+      </select>
+      <button type="submit">Save</button>
+    </form>
+  );
+}
+
+export default BookForm;
```
Comment (BrittanyBlake) [inline src/component/BookForm.js]: - [ ] The  `BookForm` component should be placed in the `src/containers` directory.

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required  🏗️ 🔨 
****
### Hello Team  👋 
You are doing a great job with this milestone! 🤩  You are almost done! 💪  There are just a few small issues that need to be resolved before you can move on: ⬇️ 
****

- [ ] It is important to follow the pattern on how to divide components into Containers and Presentational Components. Container components are the ones that are connected to the redux store, in this case, BookList and BookForm. Presentational components are the ones that do not communicate with redux. 
 Guidelines from pathwright 👇🏾 
Screen Shot 2021-02-03 at 12 54 52 AM


****
### Kindly make the required changes and resubmit for a new code review once you are ready 🙏


Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Review by shloch — APPROVED** [review]
## Hello @carlosprietobarron 
accomplished

Components relocated, we're good to go.
Great job.
HAPPY CODING

**Pull request merged**